src/node.js

```diff
@@ -340,7 +340,8 @@
     function _tickCallback() {
       var callback, threw, tock;
 
-      scheduleMicrotasks();
+      if (microtasksScheduled)
```

Comment:
Displays after 1000ms timeout. I'm going to add tests for this branch of execution.